### PR TITLE
Bug 1190536 - Locales not picked up on Pinning the web

### PR DESCRIPTION
**Pull request by albertopq**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=dc4bbc9ef839dcf7d213def0514ea3bff30e58ff

File: apps/system/locales/system.en-US.properties [hunk 1]
```diff
@@ -609,10 +609,12 @@ okCharsLeft = OK ({{n}})
...
 # Pinning the Web
 
-# LOCALIZATION NOTE (pinning.pin, pinning.pin-page): These strings are used as
-# button labels. "Pin" indicates the action to pin a website.
-pinning.pin=Pin
-pinning.pin-page=Pin Page
+# LOCALIZATION NOTE (pinning.pin): This string is used as
+# button label. "Pin" indicates the action to pin a website.
+pinning-pin=Pin
+# LOCALIZATION NOTE (pinning.pin-page): This string is used as
```
Comment (TheoChevalier) [inline apps/system/locales/system.en-US.properties]: nit: pinning-pin-page

File: apps/system/locales/system.en-US.properties [hunk 2]
```diff
@@ -609,10 +609,12 @@ okCharsLeft = OK ({{n}})
 
 # Pinning the Web
 
-# LOCALIZATION NOTE (pinning.pin, pinning.pin-page): These strings are used as
-# button labels. "Pin" indicates the action to pin a website.
-pinning.pin=Pin
-pinning.pin-page=Pin Page
+# LOCALIZATION NOTE (pinning.pin): This string is used as
```
Comment (TheoChevalier) [inline apps/system/locales/system.en-US.properties]: nit: pinning-pin

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b2f56a4f42560540300630402cf51169f5506584

**Pull request merged**